modules/ROOT/pages/functions/string.adoc

```diff
@@ -148,6 +148,173 @@ RETURN ltrim('   hello')
 
 ======
 
+
```

Comment:
I leave this entirely to your discretion, but the IS NORMALIZED section has a link to the normalize() function but not the other way around.

Reply:
Good suggestion @AzuObs

Reply:
Ah nice, I can add that in :)